src/main/java/com/odysseusinc/arachne/executionengine/service/impl/SQLServiceImpl.java

```diff
@@ -96,33 +98,36 @@ public void analyze(AnalysisRequestDTO analysis, File file, ResultCallback resul
                         final String sqlFileName = sqlFile.getName();
                         try (OutputStream outputStream = new ByteArrayOutputStream()) {
                             Files.copy(sqlFile.toPath(), outputStream);
-                            try (Statement statement = conn.createStatement();) {
-                                if (statement.execute(outputStream.toString())) {
-                                    try (ResultSet resultSet = statement.getResultSet()) {
-                                        if (resultSet != null) {
-                                            resultFile = Paths.get(sqlFile.getAbsolutePath() + ".result.csv");
-                                            try (PrintWriter out = new PrintWriter(new BufferedWriter(
-                                                    new FileWriter(resultFile.toFile(), true))
-                                            )) {
-                                                ResultSetMetaData metaData = resultSet.getMetaData();
-                                                int columnCount = metaData.getColumnCount();
-                                                for (int column = 1; column <= columnCount; column++) {
-                                                    String columnLabel = metaData.getColumnLabel(column);
-                                                    out.append(columnLabel);
-                                                    if (column < columnCount) {
-                                                        out.append(csvSeparator);
+                            List<String> sqlStatementsList = Arrays.asList(SqlSplit.splitSql(outputStream.toString()));
+                            for (int statementIdx = 0; statementIdx < sqlStatementsList.size(); statementIdx++) {
+                                try (Statement statement = conn.createStatement();) {
+                                    if (statement.execute(sqlStatementsList.get(statementIdx))) {
```

Comment:
Only one question: what about transactions? 
Previously it was run as a single statement in single transaction.
But now it seems running each statement on it's own transaction due to autoCommit mode is turned on.